mmv1/templates/terraform/constants/privileged_access_manager_entitlement.go.erb

```diff
@@ -20,4 +20,4 @@ func validateEntitlementId(v interface{}, k string) (ws []string, errors []error
     }
 
     return
-}
```

Comment:
How is this working now? If it is not working, why is there no tests that are failing?

Reply:
I see that validateEntitlementId func was not closed. Is it working now? (without adding this closing curly braces)

Reply:
The function is closed Goutham, There must be some space or newline change for which git is showing us a diff.